src/Core/ActionBar/ActionsBarButton.php

```diff
@@ -24,7 +24,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-namespace PrestaShopBundle\Component\ActionBar;
+namespace PrestaShop\PrestaShop\Core\ActionBar;
```

Comment:
Oh this one was just a suggestion but I guess it can be moved Moving straight away is a BC break though, it might be a bit rash to break this already Maybe we should have an intermediate state where we keep duplicate extending classes for backward compatibility but declare them as deprecated for now

Reply:
Like `PrestaShopBundle\Component\ActionBar\ActionsBarButton` extends `PrestaShop\PrestaShop\Core\ActionBar\ActionsBarButton`

Reply:
If I'm not wrong, these classes are new in 8.0.0
Need to check

Reply:
Oh actually they've been modified by @Progi1984 in this PR https://github.com/PrestaShop/PrestaShop/pull/25066 without marking the BC break (shame!)

So ideally you should first restore them in the initial namespace `PrestaShopBundle\Controller\Admin\Sell\Order`
https://github.com/Progi1984/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/ActionsBarButton.php

Then copy the classes in the new namespace `PrestaShop\PrestaShop\Core\ActionBar` Finally the former order classes should extend the new one and be deprecated This way modules can still use them without breaking change

In a future version we will clean the old one but at least it leaves some time for modules to update their dependency